src/sort-numbers/built-in-sort-rayane/sort-numbers-rayane.spec.js

```diff
@@ -0,0 +1,99 @@
+import { sortArrayOfNumbers } from './sort-numbers-rayane.js';
+
+describe('sortArrayOfNumbers sorts an array of numbers', () => {
+  describe('sorts positive numbers', () => {
+    it('whole numbers', () => {
+      const received = sortArrayOfNumbers([25, 45, 10, 3, 81]);
+      const expected = [3, 10, 25, 45, 81];
+      expect(received).toEqual(expected);
+    });
+    it('decimal numbers', () => {
+      const received = sortArrayOfNumbers([9.4, 0.01, 4.1, 1.11]);
+      const expected = [0.01, 1.11, 4.1, 9.4];
+      expect(received).toEqual(expected);
+    });
+    it('mixed whole and decimal numbers', () => {
+      const received = sortArrayOfNumbers([4.52, 0.2, 9.7]);
```

Comment:
here the test example only includes decimal numbers while in line 15 it's mentioned that it should include  whole and decimal numbers.

Reply:
@katsmamina Thank for your comments. I pushed my changes. Can you check it again?